### PR TITLE
Feature/support uml element titles

### LINKED ISSUE #12: Properly fill the title of a component
the diagram:
```
@startuml
interface _COMP_EXT1_IF1 as SwfComponentBase
component _COMP8 as RoadBoundaryFusion
@enduml
```

results in the formatted graph:
```
{
  "nodes": [
    {
      "name": "_COMP_EXT1_IF1",
      "members": [],
      "id": "_COMP_EXT1_IF1",
      "type": "Interface",
      "hidden": true
    },
    {
      "name": "_COMP8",
      "id": "_COMP8",
      "type": "Component",
      "title": "_COMP8",
      "hidden": true
    }
  ],
  "edges": [
    {
      "from": "_COMP8",
      "to": "",
      "name": "contains",
      "hidden": true
    }
  ]
}
```

ID, title and name of a component are identical but. But the title should be the string after `as`.

For example:
`component _COMP8 as RoadBoundaryFusion`
should result in
```
    {
      "name": "_COMP8",
      "id": "_COMP8",
      "type": "Component",
      "title": "RoadBoundaryFusion",
      "hidden": true
    }
```

### PR DESCRIPTION
**Pull request by Enteee**
fix #12

### CODE REVIEW

File: test/fixtures/component-components/parse-out.default [hunk 1]
```diff
@@ -2,13 +2,16 @@
   {
     "elements": [
       {
-        "name": "Business Logic"
+        "name": "Business Logic",
+        "title": "Business Logic"
       },
       {
-        "name": "Data Access"
+        "name": "Data Access",
```
Comment (DaelDe) [inline test/fixtures/component-components/parse-out.default]: should be DA?

```
[Data Access] as DA
```

File: test/fixtures/component-components/parse-out.default [hunk 2]
```diff
@@ -2,13 +2,16 @@
...
+        "title": "Data Access"
       },
       {
-        "name": "Graphic User\\nInterface"
+        "name": "Graphic User\\nInterface",
```
Comment (DaelDe) [inline test/fixtures/component-components/parse-out.default]: same here, name or title is GUI?

File: test/fixtures/component-components/parse-out.default (hunk 1 above)
Comment (Enteee) [reply test/fixtures/component-components/parse-out.default]: you are right, I forgot to implement the title feature for the short syntax for components/use-cases. Will fix.

Comment (Enteee) [reply test/fixtures/component-components/parse-out.default]: this should be fixed now.

File: test/fixtures/component-components/parse-out.default (hunk 2 above)
Comment (Enteee) [reply test/fixtures/component-components/parse-out.default]: this should be fixed now.

**Comment by Enteee** [comment]
no response from @DaelDe,  assuming this looks all god.

**Pull request merged**